test/ad/distributions.jl

```diff
@@ -435,17 +433,21 @@
             # Skellam only fails in these tests with ReverseDiff
             # Ref: https://github.com/TuringLang/DistributionsAD.jl/issues/126
             # PoissonBinomial fails with Zygote
+            # Matrix case does not work with Skellam:
+            # https://github.com/TuringLang/DistributionsAD.jl/pull/172#issuecomment-853721493
             filldist_broken = if d.f(d.θ...) isa Skellam
-                (d.broken..., :ReverseDiff)
+                ((d.broken..., :ReverseDiff), (d.broken..., :Zygote, :ReverseDiff))
```

Comment:
Is the redundant `:ReverseDiff` in `(d.broken..., :ReverseDiff)` and `(d.broken..., :Zygote, :ReverseDiff)` intended?

Reply:
Yes, ReverseDiff still doesn't work for both tested sizes (single sample + two samples) whereas Zygote only fails if there are multiple samples. Upstream changes in SpecialFunctions and Zygote fixed most of the issues in https://github.com/TuringLang/Bijectors.jl/issues/156 (and https://github.com/TuringLang/DistributionsAD.jl/issues/145) but this one is still present.